next.config.js

```diff
@@ -27,6 +27,7 @@ const nextConfig = {
       "www.frenslands.xyz",
       "raw.githubusercontent.com",
       "lh3.googleusercontent.com",
+      "i.ibb.co"
```

Comment:
please move sources inside the repo, see other dapp's json

Reply:
maybe I didn't get it correct, this is where I have added the outer sources into next config file
https://github.com/argentlabs/dappland/pull/18/commits/c4528a8e12168d7a1d12bd2ab532c33eab74478e

Reply:
right, but that's not the case anymore
Please rebase your branch and see others, like here:
https://github.com/argentlabs/dappland/blob/develop/data/aspect.json#L40

Static files are now inside the repo